src/app/dashboard-layout/dashboard-layout.component.html

```diff
@@ -1,4 +1,17 @@
-<p>dashboard-layout works!</p>
-<app-header></app-header>
-<app-courses-list></app-courses-list>
-<app-footer></app-footer>
+<div class="dashboard-layout">
+  <app-header></app-header>
+
+  <div class="breadcrumb">
```

Comment:
use dashboard layout as a dumb component to show footer and header. with ng-content inside
CourseList should be routed smart